README.md

```diff
@@ -2,62 +2,123 @@
 
 # End to end tests for Archive.org using [Playwright](https://playwright.dev/)
 
+
 ## (Optional) BrowserStack Local Setup
 
-- login to BrowserStack and retrieve the account username and access key, see documentation [here](https://www.browserstack.com/docs/automate/playwright/getting-started/nodejs/test-runner)
+- Login to BrowserStack and retrieve the account username and access key, see documentation [here](https://www.browserstack.com/docs/automate/playwright/getting-started/nodejs/test-runner)
 
 
-## Running tests locally
+## Local Setup
 
-- install dependencies:
+- Install dependencies:
 
     `npm i`
 
-- install Playwright browser libs:
+- Install Playwright browser libs:
 
     `npx playwright install`
 
-- run all the tests in headless mode and generate 1 whole test report:
+- Create a `.env` file by copying the contents from `.env.sample` and add the respective values you want to use for testing
+
+    - this is required to run tests with loggedIn flows
+
+
+## Running tests
+
+- Run all tests: `npm run test`
+
+- Run tests using `generate` command:
+
+    - See `scripts/generateCommand.js`, update the arguments that starts at `line:36`
+
+    - Run the command: `npm run generate:command`, run the generated by the command
```

Comment:
QUESTION: Is there a missing word here? "run the ??? generated by the command"? It's not clear what `generate` does here. Maybe add a couple examples?